README.md

```diff
@@ -54,9 +54,18 @@ First get all the requirements installed on your system.
 ### Prerequisites
 
 - [Node.js LTS (16.x)](https://nodejs.org/)
-- [Python 3.8.x](https://python.org/) or
+- [Python](https://python.org/) - See below for specific versions.
 - [Docker](https://www.docker.com/get-started) (if using the Plone docker images)
 
+The versions of Python that are supported in Volto depend on the version of Plone that you use.
+
+| Plone | Python | Volto |
+|---|---|---|
+| 5.2 | 2.7, 3.6-3.8 | 14.0(? or earlier?)-15.0 |
```

Comment:
Probably easiest to just recommend Volto 15. I am not sure about earlier versions, since I just came back in June. @sneridagh?

Reply:
Done.